tests/baselines/reference/tsc/runWithoutArgs/initial-build/does-not-add-color-when-NO_COLOR-is-set.js

```diff
@@ -68,16 +68,18 @@ default: true
 
 --target, -t
 Set the JavaScript language version for emitted JavaScript and include compatible library declarations.
-one of: es3, es5, es6, es2015, es2016, es2017, es2018, es2019, es2020, es2021, esnext

```

Comment:
I'm guessing the non-relevant changes in this file are going to go away if we merge from main?

Reply:
maybe not though